Error: Invalid format '├── https://github.com/sponsors/broofa'

Getting the error listed above at the tail end of execution.  The entire job seems to be running successfully (i.e. there is a message that says "✔  success   Version bumped!"), but then there is an error about being unable to process file command 'output' successfully.

Full log:

```
process.env.GITHUB_WORKSPACE /home/runner/work/<appname>/<appname>/apps/<msname>
tagPrefix: 
tagSuffix: 
commit messages: [
  'force build\nundefined',
  'Merge pull request #1348 from sernaferna/bnd\n\nforce build\nundefined'
]
config words: {
  majorWords: [ 'MAJOR' ],
  minorWords: [ 'MINOR' ],
  patchWords: null,
  preReleaseWords: [ 'pre-alpha', 'pre-beta', 'pre-rc' ]
}
version action after first waterfall: patch
version action after final decision: patch
runInWorkspace | command: git args: [ 'config', 'user.name', '"Automated Version Bump"' ]
runInWorkspace | command: git args: [
  'config',
  'user.email',
  '"gh-action-bump-version@users.noreply.github.com"'
]
currentBranch: main
runInWorkspace | command: npm args: [
  'version',
  '--allow-same-version=true',
  '--git-tag-version=false',
  '6.1.5'
]
current 1: 6.1.5 / version: patch
newVersion 1:   run `npm fund` for details
runInWorkspace | command: git args: [
  'commit',
  '-a',
  '-m',
  'ci: version bump to   run `npm fund` for details'
]
runInWorkspace | command: git args: [ 'checkout', 'main' ]
runInWorkspace | command: npm args: [
  'version',
  '--allow-same-version=true',
  '--git-tag-version=false',
  '6.1.5'
]
current 2: 6.1.5 / version: patch
execute npm version now with the new version: patch
newVersion 2:   run `npm fund` for details
newVersion after merging tagPrefix+newVersion+tagSuffix:   run `npm fund` for details
runInWorkspace | command: sh args: [
  '-c',
  'echo "newTag=  run `npm fund` for details" >> $GITHUB_OUTPUT'
]
runInWorkspace | command: git args: [
  'commit',
  '-a',
  '-m',
  'ci: version bump to   run `npm fund` for details'
]
git commit failed because you are using "actions/checkout@v2" or later; but that doesnt matter because you dont need that git commit, thats only for "actions/checkout@v1"
runInWorkspace | command: git args: [
  'push',
  '***github.com/sernaferna/<appname>.git'
]
✔  success   Version bumped!
##[error]Unable to process file command 'output' successfully.
##[error]Invalid format '├── https://github.com/sponsors/broofa'
```

*There is an error about a git commit failing, too, but I don't* think *that's causing the problem -- I could be wrong.*

This is for a smaller microservice within a larger monorepo; the relevant section of the YAML is:

```yaml
  bump-versions:
    needs: paths-filter
    name: Bump version numbers
    runs-on: ubuntu-latest

    steps:
      - name: Checkout source code
        uses: actions/checkout@v4
        with:
          ref: ${{ github.ref }}

      - name: Bump version for <msname>
        uses: phips28/gh-action-bump-version@master
        if: ${{ needs.paths-filter.outputs.msname == 'true' }}
        with:
          target-branch: 'main'
          skip-tag: 'true'
          major-wording: 'MAJOR'
          minor-wording: 'MINOR'
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          PACKAGEJSON_DIR: 'apps/msname'
          GITHUB_REF: ${{ github.ref}}

      - name: Bump version for Workspace
        uses: phips28/gh-action-bump-version@master
        with:
          target-branch: 'main'
          major-wording: 'MAJOR'
          minor-wording: 'MINOR'
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          GITHUB_REF: ${{ github.ref}}
```

I'm not doing an `npm install` anywhere before this script runs; my YAML does the following:

1. Job to get changed apps (further jobs use this output to only bump/build/deploy changed apps)
    1. `checkout@v4`
    2. `dorny/paths-filter@v3`
2. Job to Bump versions
    1. `checkout@v4`
    2. `phips28/gh-action-bump-version`
3. Job to build and deploy App 1
4. Job to build and deploy App 2
5. ...

As for the new version, it doesn't work with **npm workspaces**.

```
npm ERR! code ENOWORKSPACES
npm ERR! This command does not support workspaces.

npm ERR! A complete log of this run can be found in: /home/runner/.npm/_logs/2024-03-15T15_21_29_725Z-debug-0.log
✖  fatal     Error: Command failed: npm config set fund false
npm ERR! code ENOWORKSPACES
npm ERR! This command does not support workspaces.

npm ERR! A complete log of this run can be found in: /home/runner/.npm/_logs/2024-03-15T15_21_29_725Z-debug-0.log

    at checkExecSyncError (node:child_process:890:11)
    at execSync (node:child_process:962:15)
    at /home/runner/work/_actions/phips28/gh-action-bump-version/turn-off-npm-fund/index.js:201:5
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
✖  fatal     Failed to bump version
```

Unfortunately it still doesn't work in a workspace:

```
process.env.GITHUB_WORKSPACE /home/runner/work/soulhx/soulhx/apps/papi
tagPrefix: 
tagSuffix: 
commit messages: [
  'new attempt for bumping\nundefined',
  'Merge pull request #1437 from sernaferna/bump\n' +
    '\n' +
    'new attempt for bumping\n' +
    'undefined'
]
config words: {
  majorWords: [ 'MAJOR' ],
  minorWords: [ 'MINOR' ],
  patchWords: null,
  preReleaseWords: [ 'pre-alpha', 'pre-beta', 'pre-rc' ]
}
version action after first waterfall: patch
version action after final decision: patch
runInWorkspace | command: git args: [ 'config', 'user.name', '"Automated Version Bump"' ]
runInWorkspace | command: git args: [
  'config',
  'user.email',
  '"gh-action-bump-version@users.noreply.github.com"'
]
currentBranch: main
runInWorkspace | command: npm args: [ 'config', 'set', 'fund', 'false' ]
✖  fatal     npm ERR! code ENOWORKSPACES
npm ERR! This command does not support workspaces.
npm ERR! A complete log of this run can be found in: /home/runner/.npm/_logs/20[24](https://github.com/sernaferna/soulhx/actions/runs/8308005489/job/22737785808#step:5:25)-03-16T13_08_[34](https://github.com/sernaferna/soulhx/actions/runs/8308005489/job/22737785808#step:5:35)_978Z-debug-0.log
npm exited with code 1
✖  fatal     Failed to bump version
```

--- a/index.js
+++ b/index.js
@@ -198,7 +198,7 @@ const pkg = getPackageJson();
     }
 
     // disable npm fund message, because that would break the output
-    await runInWorkspace('npm', ['config set fund', 'false']);
+    await runInWorkspace('npm', ['config', 'set fund', 'false']);
 
     // do it in the current checked out github branch (DETACHED HEAD)
     // important for further usage of the package.json version

--- a/index.js
+++ b/index.js
@@ -198,7 +198,7 @@ const pkg = getPackageJson();
     }
 
     // disable npm fund message, because that would break the output
-    await runInWorkspace('npm', 'config set fund false');
+    await runInWorkspace('npm', ['config set fund', 'false']);
 
     // do it in the current checked out github branch (DETACHED HEAD)
     // important for further usage of the package.json version

--- a/index.js
+++ b/index.js
@@ -198,7 +198,7 @@ const pkg = getPackageJson();
     }
 
     // disable npm fund message
-    execSync('npm config set fund false --location=global');
+    execSync('npm config set fund false');
 
     // do it in the current checked out github branch (DETACHED HEAD)
     // important for further usage of the package.json version

--- a/index.js
+++ b/index.js
@@ -198,7 +198,7 @@ const pkg = getPackageJson();
     }
 
     // disable npm fund message, because that would break the output
-    await runInWorkspace('npm', ['config', 'set fund', 'false']);
+    await runInWorkspace('npm', ['config', 'set', 'fund', 'false']);
 
     // do it in the current checked out github branch (DETACHED HEAD)
     // important for further usage of the package.json version

--- a/index.js
+++ b/index.js
@@ -197,6 +197,9 @@ const pkg = getPackageJson();
       return;
     }
 
+    // disable npm fund message
+    execSync('npm config set fund false --location=global');
+
     // do it in the current checked out github branch (DETACHED HEAD)
     // important for further usage of the package.json version
     await runInWorkspace('npm', ['version', '--allow-same-version=true', '--git-tag-version=false', current]);

--- a/index.js
+++ b/index.js
@@ -198,7 +198,8 @@ const pkg = getPackageJson();
     }
 
     // disable npm fund message, because that would break the output
-    await runInWorkspace('npm', ['config', 'set', 'fund', 'false']);
+    // -ws/iwr needed for workspaces https://github.com/npm/cli/issues/6099#issuecomment-1961995288
+    await runInWorkspace('npm', ['config', 'set', 'fund', 'false', '-ws=false', '-iwr']);
 
     // do it in the current checked out github branch (DETACHED HEAD)
     // important for further usage of the package.json version

--- a/index.js
+++ b/index.js
@@ -197,8 +197,8 @@ const pkg = getPackageJson();
       return;
     }
 
-    // disable npm fund message
-    execSync('npm config set fund false');
+    // disable npm fund message, because that would break the output
+    await runInWorkspace('npm', 'config set fund false');
 
     // do it in the current checked out github branch (DETACHED HEAD)
     // important for further usage of the package.json version

